brightnessd.c

```diff
@@ -1027,7 +1114,14 @@ static uint8_t event_loop(struct Tglobalstate *pglobalstate, struct Txcb *pxcb,
 
     @see event_loop
 */
-int main() {
+int main (int argc, char** argv) {
+
+    if (parse_args(argc, argv)) {
+        ERROR("[main] Error parsing command-line arguments.\n");
+        return 1;
```

Comment:
Nitpicking: this should be done by `exit(EXIT_FAILURE);` to make it more explicit

Reply:
Fixed